pkg/sql/codegen/pai/codegen.go

```diff
@@ -122,20 +122,24 @@ func Train(ir *ir.TrainStmt, session *pb.Session, tarball, paramsFile, modelName
 			return
 		}
 	} else if strings.HasPrefix(strings.ToLower(ir.Estimator), "xgboost") {
-		ossURI := OSSModelURL(ossModelPath)
-		if code, e = xgboost.DistTrain(ir, session, cc.Worker.Count, ossURI); e != nil {
+		ossURIToSave := OSSModelURL(ossModelPathToSave)
+		ossURIToLoadPreTrainedModel := ""
```

Comment:
Simplify to `ossURIToLoad`?

Reply:
Check if `ossModelPathToSave == ossModelPathToLoadPreTrained`?

Reply:
> Simplify to ossURIToLoad?
Done.

> Check if ossModelPathToSave == ossModelPathToLoadPreTrained?
There is no need to check. We can support both `save_path == load_path` and `save_path != load_path`.

Reply:
I see.